.pre-commit-config.yaml

```diff
@@ -0,0 +1,26 @@
+---
+
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.0.1  # Use the ref you want to point at
```

Comment:
oh could this point to the newest version please `v4.1.0`

Reply:
:+1: